compile/service.go

```diff
@@ -62,11 +63,18 @@ func compileService(file string, src *ast.Service) (*ServiceSpec, error) {
 		functions[function.Name] = function
 	}
 
+	annotations, err := compileAnnotations(src.Annotations)
+
+	if err != nil {
+		return nil, err
```

Comment:
This should probably be a `compileError` too to include the service name in the error.